playwright-report/index.html

```diff
@@ -0,0 +1,62 @@
+
```

Comment:
@grantnorwood looks like this was leftover from another branch, mind deleting?

Reply:
Ugh, me working on 3-4 branches at the same time 🤦   Yup, I'll pull that outta this one.